moto/cloudformation/utils.py

```diff
@@ -134,3 +136,29 @@ def get_stack_from_s3_url(template_url: str, account_id: str, partition: str) ->
 
     key = s3_backends[account_id][partition].get_object(bucket_name, key_name)
     return key.value.decode("utf-8")  # type: ignore[union-attr]
+
+
+def validate_create_change_set(
+    stack_name: str,
```

Comment:
Can we change this here to only include the arguments that we're actually using?
The less arguments, the easier it is to read - and we can always change the signature later